programs/drift/src/math/matching.rs

```diff
@@ -17,6 +17,11 @@ pub fn is_maker_for_taker(
     taker_order: &Order,
     slot: u64,
 ) -> DriftResult<bool> {
+    // Maker and taker order not allowed to both be placed in the current slot
```

Comment:
`// Maker and taker order not allowed to match if both were placed in the current slot`